cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -942,8 +942,8 @@
   </property>
 
   <property>
-    <name>log.saver.run.memory.megs</name>
-    <value>1024</value>
+    <name>log.saver.run.memory.mb</name>
+    <value>${master.service.memory.mb}</value>
```

Comment:
This is explicitly set higher because the Log Saver requires more memory than the Master service. Also, you changed the property name without deprecating it, properly. 👎